src/ameba.ts

```diff
@@ -27,23 +28,68 @@ export class Ameba {
         this.config = getConfig();
     }
 
-    public execute(document: TextDocument): void {
+    public execute(document: TextDocument, virtual: boolean = false): void {
```

Comment:
Redundant type definition.

```suggestion
    public execute(document: TextDocument, virtual = false): void {
```

Reply:
I guess yeah, I always like to add types even if they're inferred.